src/data/image_upload.ts

```diff
@@ -12,8 +12,14 @@ export interface ImageMutableParams {
   name: string;
 }
 
-export const generateImageThumbnailUrl = (mediaId: string, size: number) =>
-  `/api/image/serve/${mediaId}/${size}x${size}`;
+export const generateImageThumbnailUrl = (
+  mediaId: string,
+  size: number,
```

Comment:
size should be optional now that original can be set, if original is false size should be set.